### DISCUSSION TITLE
Theming like Glass8

### DISCUSSION THREAD

**Discussion by XLRUUU** [Maplespe/DWMBlurGlass#205] [Ideas]
Would it be possible to add a functionality like the Glass8 ThemeAtlas which allows to theme window title bar and window borders with an image file?

I found Windows 7/Vista themes for Windows 10, but it seems like there's no theme for Windows 11, so it could be a easier way to theme window apparence.

**Comment by MagicAndre1981** [comment]
I request this, too.
